### ISSUE TITLE
CSV

### ISSUE THREAD

**Issue by MrHamel** [docjoel/youtubemissingvideonotify#1]
While the CSV file format is a very simple and easy to use database, but you will run into video title conflicts as they will add additional columns to the row.

**Comment by docjoel** [comment]
Yes, I think I fixed this with some formatting and will go in later, to with the Regex

**Comment by docjoel** [comment]
I also am having the API generate the CSV from the JSON. I want an end product that is human readable.

**Comment by MrHamel** [comment]
With this project, is the goal to make some sort of desktop app, or is this going to be a web app?

**Comment by docjoel** [comment]
The end goal is to make work for either. I plan on introducing a web app version using a scheduler like Huey so I can do daily checks.
